src/pages/Gallery.js

```diff
@@ -41,87 +41,119 @@ const StyledGallery = styled.section`
 `;
 
 const StyledRadioContainer = styled.div`
-    position: absolute;
-    bottom: 2rem;
-    left: 2rem;
-    
-    &>input{
-        opacity: 1;
-        margin: 1rem;
-        cursor: pointer;
-        
-        
-
-        
-        &::after {
-            content: '';
-            display: block;
-            width: 20px;
-            height: 20px;
-            background-color: green;
-            border-radius: 50%;
-            opacity: 0.4;
-            transform: translate(-4px, -34px);
-            z-index: 10;
-        }
-
-        &::before {
-            content: '';
-            display: block;
-            width: 30px;
-            height: 30px;
-            background-color: lightcoral;
-            opacity: 0;
-            border-radius: 50%;
-            transform: translate(-9px, -9px);
-            z-index: 5;
-        }
-
-        &:checked::before {
-            opacity: 0.4;
-            
-        }
-    }
-    
-`
+  position: absolute;
+  bottom: 2rem;
+  left: 2rem;
+`;
+
+const StyledInput = styled.input`
+  margin: 1rem;
+  cursor: pointer;
+
+  &::after {
+    content: "";
+    display: block;
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    opacity: 1;
+    transform: translate(-4px, -34px);
+    z-index: 10;
+  }
+
+  &:nth-child(1)::after {
+    background-color: black;
+  }
+
+  &:nth-child(2)::after {
+    background-color: red;
+  }
+
+  &:nth-child(3)::after {
+    background-color: grey;
+  }
+
+  &:nth-child(4)::after {
+    background-color: #3d5367;
+  }
+  &::before {
+    content: "";
+    display: block;
+    width: 30px;
+    height: 30px;
+    background-color: white;
+    opacity: 0;
+    border-radius: 50%;
+    transform: translate(-9px, -9px);
+    z-index: 5;
+  }
+
+  &:checked::before {
+    opacity: 1;
+  }
+`;
 
 const Gallery = () => {
-  const [radio, setRadio] = useState("color1");
+  const [radio, setRadio] = useState("");
+  const [img, setImg] = useState("images/photo1.png");
+  // const isSelected = (value) => true;
+
+  useEffect(() => {
+    if (radio === "red") {
+      setImg("images/photo5.png");
```

Comment:
maybe change filenames to photo_red, photo_blue...
```suggestion
      setImg("images/photo5.png");
```
setImg(`images/photo_${radio}`);